### ISSUE TITLE
NTR: CT, MRI or angiography: pancreas vascular malformation (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8970]
New term request CT, MRI or angiography: pancreas vascular malformation (elthat_121129182940): Pancreatic arteriovenous malformation; AVM; hemangiomas

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000379
- label: Pancreatic arteriovenous malformation
- parents: [Abnormal pancreas morphology (HP:0012090)](https://hpo.jax.org/app/browse/term/HP:0012090)
- pmid: PMID:29904461
- def: Pancreatic arteriovenous malformation is defined as an abnormal vascular net, developed from one or more feeding arteries and enlarged early draining veins, creating an arteriovenous shunting.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8970
- synonyms:
